src/ledgermonolith/init/startup-script.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Startup script to start the ledgermonolith service from a JAR.
+#
+# Expects build artifacts to be available on Google Cloud Storage at
+# gs://bank-of-anthos/monolith.
```

Comment:
This is problematic because if we're providing instructions for any user to `make monolith` and push to their project, hardcoding the storage bucket to the release bucket doesn't make sense.

We should create a dynamic storage bucket name based on the PROJECT_ID.